meerk40t/gui/wxmeerk40t.py

```diff
@@ -3511,7 +3511,7 @@ def set_icon(self, node, icon=None):
 
     def update_label(self, node, force=False):
         if node.label is None or force:
-            node.label = node.label_from_source_cascade()
```

Comment:
This doesn't seem like it would work. The reason we force this is so that demands an update. It seems like it won't cascade through the various methods without this.

Reply:
Also, why is it using set_label like it gets a label?

Reply:
This seems like it prevents the label updating in situations where "force" would be called. The flag there is by design since sometimes the label would already be set but a change means it needs to update.

Reply:
I agree that the method is a bit confusing. We can easily create:
```
    def set_label(self, name):
        """
        Set the name of this node to the name given.
        :param name: Name to be set for this node.
        :return:
        :return: label
        """
        self.label = self.create_label(name)

     def create_label(self, name=None):
        if name is not None:
            return name
        if self.object is not None:
            return self.label_from_source_cascade(self.object)
        return str(self)
```

Reply:
Then we can use create_label in wxmeerk40t.

Reply:
I also am not sure this fixes the force which is required when update is called to force the label to relabel.